pytrip/res/concave_tool_v2.py

```diff
@@ -0,0 +1,111 @@
+"""

```

Comment:
change filename to something more contour generic.
This algorithm calculates not only concave contours, but all, so filename doesn't reflect what it does.
Newcommer will not have a clue why this name.
I also would delete `tool` part here and in intersection_tool